src/hooks/useDialogArea.tsx

```diff
@@ -10,7 +10,7 @@ export const DialogAreaProvider = ({ ...props }) => (
   <DialogAreaContext.Provider value={useDialogAreaContext()} {...props} />
 );
 
-export const useDialogArea = () => useContext(DialogAreaContext)!;
```

Comment:
i'm removing the return type casting as the code impact is marginal + i feel strongly that it's best practice to code around types instead of vice versa